data-table/__tests__/table.js

```diff
@@ -4,6 +4,113 @@ beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
+describe('basic tests to ensure createDataTable can function well', () => {
+    beforeEach(() => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id'
+        });
+    });
+
+    test('check the add column button', () => {
+        const config = {
+            'wrapperDivId': 'div-id',
+        };
+        table.createDataTable(config);
+        const contents = document.getElementById(config.wrapperDivId).textContent;
+        expect(contents.substr(0, 12)).toEqual("+ Add column");
+    });
+
+    test('check the default value of rows', () => {
+        const contents = document.getElementsByTagName('table')[0].rows.length;
```

Comment:
The variable name `contents` is inaccurate - this is a length, not actual contents. Consider `const numRows = ...`